src/movement/thruster_sabertooth.cpp

```diff
@@ -22,6 +22,49 @@ void checksum (uint8_t serial_data[4])
   serial_data[3] = (serial_data[0]+serial_data[1]+serial_data[2]) & 127;
 }
 
+void createThrusterPacket (uint8_t serial_data[4], double value, int i)
+{
+    serial_data[0] = mThruster_info[i].address;
+    //command
+    if(mThruster_info[i].port == 0)
+    {
+      if(value < 0)
+      { //command - backwards port 1
+        serial_data[1] = 1;
+      }
+      else
+      { //command - forwards port 1
+        serial_data[1] = 0;
+      }
+    }
+    else
+    {
+      if(value < 0)
```

Comment:
What is the difference between the two cases of `mThruster_info[i].port ==0` and the else? Your commments seem to lead me to believe that there is no difference.